src/util/mod.rs

```diff
@@ -28,3 +28,7 @@ pub use self::rng::reset_rng_state;
 
 #[cfg(test)]
 pub(crate) mod test_utils;
+
+pub use self::configurations::UserConfigurations;
```

Comment:
I'm still not sure why this is re-exported. This seems very inconsistent with other parts of the lib. In the future we can revisit re-exporting things at the top of the crate in line with what other rust libs do, but that's probably orthogonal to this PR and for now we should keep things consistent.